src/main/scala/faces/utils/Helpers.scala

```diff
@@ -126,10 +129,10 @@ case class Helpers(cfg: FacesSettings)(implicit rnd: Random) {
     new File(bgPath).listFiles.filter(_.getName.endsWith(bgType)).toIndexedSeq
   }
 
-  def writeLandmarks(rps: RenderParameter, file: File): Try[Unit] = {
+  def writeLandmarks(rps: RenderParameter, file: File, mask:PixelImage[RGBA]): Try[Unit] = {
```

Comment:
please keep a writeLandmarks without mask that automatically generates a mask and then calls writelandmarks with mask.